### ISSUE TITLE
Help:  Implementing this LSP on emacs.  (Some errors happening)

### ISSUE THREAD

**Issue by otavioschwanck** [qiuxiang/solidity-ls#1]
Hi,

i am implementing this LS on emacs and need some help.

When i try to execute on terminal:

```
➜  nest-onchain git:(feature/investment-withdrawal) ✗ npx solidity-ls --stdio

/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 1: var: command not found
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 2: var: command not found
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 3: var: command not found
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 4: var: command not found
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 5: var: command not found
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 6: var: command not found
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 7: syntax error near unexpected token `('
/Users/otavio/.npm/_npx/67117/bin/solidity-ls: line 7: `var __export = (target, all) => {'
```

Its a bug?  Here, on coc its working.

Thanks

**Comment by qiuxiang** [comment]
u can try `npm i solidity-ls -g` and run `solidity-ls`

**Comment by qiuxiang** [comment]
Fixed in v0.4.1
